src/sound_openal.cpp

```diff
@@ -449,7 +528,7 @@ class OpenALSoundManager: public ISoundManager
 	bool loadSoundFile(const std::string &name,
 			const std::string &filepath)
 	{
-		SoundBuffer *buf = loadOggFile(filepath);
+		SoundBuffer *buf = load_ogg_from_file(filepath);
 		if(buf)
```

Comment:
Fix this to be consistent with the `loadSoundData` one (space after `if`)?  You've already fixed spaces in other places.